easybuild/easyconfigs/p/PySide6/PySide6-6.5.0-GCCcore-12.2.0.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'PythonPackage'
+
+name = 'PySide6'
+version = '6.5.0'
+
+homepage = 'https://pypi.python.org/pypi/PySide6'
+description = """PySide6 is the official Python module from the Qt for Python project,
+    which provides access to the complete Qt 6.5+ framework."""
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+source_urls = ['https://download.qt.io/official_releases/QtForPython/%(namelower)s/%(name)s-%(version)s-src/']
+sources = ['pyside-setup-everywhere-src-%(version)s.tar.xz']
+checksums = ['6ef53b29127267e381917e6f9399ce760edc0644cd5831988b1de72c9d583ab4']
+
+builddependencies = [
+    ('binutils', '2.39'),
+    ('CMake', '3.24.3'),
+    ('Clang', '15.0.7'),
```

Comment:
Must this use clang? I would expect the normal gcc toolchain to be used.

Reply:
CLang is absolutely needed at build time only for Shiboken6 wrapper.